packages/web-components/fast-components/src/tabs/tabs.styles.ts

```diff
@@ -30,11 +30,13 @@ export const TabsStyles = css`
     }
 
     .start {
```

Comment:
You can combine these selectors to reduce this:

```
.start, .end {
   padding: 2px;
   align-self: center;
}
```

Reply:
you are correct. thanks!!

Reply:
combined and also removed the padding